### ISSUE TITLE
Dependencies

### ISSUE THREAD

**Issue by Tsutsukakushi** [AlwaysGeeky/Vox#30]
It would be nice if the README listed the dependencies so people could actually compile Vox.

**Comment by AlwaysGeeky** [comment]
Hi, the dependencies should already be included as stand alone files in the directories.

Are you having trouble compiling on a particular platform??

Could you tell me what your issue is and I will try to help resolve it.

**Comment by Tsutsukakushi** [comment]
This is what I get. When trying to compile on GNU+Linux.

```
Scanning dependencies of target Vox
[  0%] Building CXX object source/CMakeFiles/Vox.dir/main.cpp.o
In file included from /home/miyu/build/Vox/source/Player/../blocks/ChunkManager.h:22:0,
                 from /home/miyu/build/Vox/source/Player/Player.h:19,
                 from /home/miyu/build/Vox/source/VoxGame.h:22,
                 from /home/miyu/build/Vox/source/main.cpp:15:
/home/miyu/build/Vox/source/Player/../blocks/Chunk.h:22:32: fatal error: ../Renderer/Camera.h: No such file or directory
compilation terminated.
source/CMakeFiles/Vox.dir/build.make:62: recipe for target 'source/CMakeFiles/Vox.dir/main.cpp.o' failed
make[2]: *** [source/CMakeFiles/Vox.dir/main.cpp.o] Error 1
CMakeFiles/Makefile2:141: recipe for target 'source/CMakeFiles/Vox.dir/all' failed
make[1]: *** [source/CMakeFiles/Vox.dir/all] Error 2
Makefile:83: recipe for target 'all' failed
make: *** [all] Error 2
```

**Comment by AlwaysGeeky** [comment]
Ahhh thanks, that is a filename casing issue... It needs to include camera.h, not Camera.h... This isn't a problem on windows, only Linux.. I will look at fixing this shortly, or you can modify your code and see if that helps.

Thanks.

**Comment by AlwaysGeeky** [comment]
I have actually fixed this issue in the repo now, if you want to pull the latest.

Fixed in commit: da20af28017429e74a12d72119569ea9a90c6e31

Let me know if you have any other issues.
